github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help-triggerMode.html

```diff
@@ -1,3 +1,9 @@
 <div>
-    Trigger modes, currently only "CRON" implemented.
+    Trigger modes.
+    <ul>
+        <li><strong>Cron</strong>: Checks for PRs to build periodically.</li>
+        <li><strong>Hooks</strong>: Listens for GitHub webhook events.</li>
+        <li><strong>Hooks plus Cron</strong>: Not fully supported - combines webhooks and cron checks.</li>
```

Comment:
Sounds like enum can be splitted to single (checkbox? :D) features and combined.